src/main/scala/com/campudus/tableaux/helper/JsonUtils.scala

```diff
@@ -42,13 +41,19 @@ object JsonUtils extends LazyLogging {
                   .flatMap(tryCast[A])
             })
         )
-    }).getOrElse(OkArg(Seq.empty))
+      })
+      .getOrElse(OkArg(Seq.empty))
   }
 
-  private def checkNotNullArray(json: JsonObject, field: String): ArgumentCheck[JsonArray] = notNull(json.getJsonArray(field), field)
+  private def checkNotNullArray(json: JsonObject, field: String): ArgumentCheck[JsonArray] =
+    notNull(json.getJsonArray(field), field)
 
   private def checkForJsonObject(seq: Seq[JsonObject]): ArgumentCheck[Seq[JsonObject]] = {
-    tryMap((y: Seq[JsonObject]) => y map { x: JsonObject => x }, InvalidJsonException(s"Warning: Array should only contain JsonObjects", "object"))(seq)
+    tryMap((y: Seq[JsonObject]) =>
+             y map { x: JsonObject =>
```

Comment:
Looks weird here. Could be changed to `y.map(_.asInstanceOf[JsonObject])` I suppose to get around this formatting issue, but unsure about it.